sdk/identity/azure-identity/CHANGELOG.md

```diff
@@ -1,8 +1,9 @@
 # Release History
 
-## 1.12.0-beta.2 (Unreleased)
+## 1.12.0 (2024-04-08)
 
 ### Features Added
+- Added default sign-in experience for brokered authentication. [#39284](https://github.com/Azure/azure-sdk-for-java/pull/39284) 
```

Comment:
This is identity-broker specific right ?
Should it be mentioned here ?

Reply:
Eh, yes, but also I think it's fine to call out here since if anyone is reading changelogs for identity news it's this one and not the broker package. Will remove it if you think it is really that out of place.